Found errors when linting GITHUB_ACTIONS. Exit code: 1.

```
GITHUB_ACTIONS
  2025-12-29 14:26:57 [INFO]   Linting GITHUB_ACTIONS items...
  Error: -29 14:26:57 [ERROR]   Found errors when linting GITHUB_ACTIONS. Exit code: 1.
  2025-12-29 14:26:58 [INFO]   Command output for GITHUB_ACTIONS:
  ------
  .github/workflows/fetch-daily.yml:41:9: shellcheck reported issue in this script: SC2086:info:2:31: Double quote to prevent globbing and word splitting [shellcheck]
     |
  41 |         run: |
     |         ^~~~
  .github/workflows/fetch-daily.yml:41:9: shellcheck reported issue in this script: SC2086:info:4:30: Double quote to prevent globbing and word splitting [shellcheck]
     |
  41 |         run: |
     |         ^~~~
  ------
```

--- a/.github/workflows/fetch-daily.yml
+++ b/.github/workflows/fetch-daily.yml
@@ -40,9 +40,9 @@ jobs:
         id: changes
         run: |
           if git diff --quiet docs/; then
-            echo "has_changes=false" >> $GITHUB_OUTPUT
+            echo "has_changes=false" >> "$GITHUB_OUTPUT"
           else
-            echo "has_changes=true" >> $GITHUB_OUTPUT
+            echo "has_changes=true" >> "$GITHUB_OUTPUT"
           fi
 
       - name: Commit and push changes

